tests/main/test_launch.py

```diff
@@ -484,7 +484,8 @@ def test_main__full_launch_script_runs_as_expected(fully_running_app_from_main_e
 
     # assert socketio was set up correctly
     ws_queue = app_info["object_access_inside_main"]["process_manager"].queue_container.to_server
-    mocked_set_up.assert_called_once_with(ws_queue)
+    to_pm_queue = app_info["object_access_inside_main"]["process_manager"].queue_container.from_websocket
```

Comment:
When grabbing queues from the queue container, you should try to use the same name so its' easier to follow. I know the line above doesn't do this either, so it would be good to fix that if you come across it anywhere